src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1625,6 +1625,28 @@ public async Task GetAsync_UnicodeHostName_SuccessStatusCodeInResponse()
             }
         }
 
+        [Fact]
+        public async Task GetAsync_CustomTransferEncoding()
```

Comment:
Nit:
```
GetAsync_CustomTransferEncoding_Succeeds
```
or something like that

Reply:
Done.

Reply:
I used `_Success` to make it consistent with the other methods.